Tidied the Install from ansible file. Added Notice for AKSK and Profile Creation from Dashboard (#76)
sodafoundation/documentation: content/soda-gettingstarted/installation-using-ansible.md

---
title: Installation Guide using Ansible

> description: "This document describes how to install SODA projects in a local cluster with detailed configuraiton options. These steps will help you to enable / disable projects. After installation using these steps, you can get the features of all the enabled projects. You can test either through SODA Dashboard UI or CLI"

Reviewer: This text is duplicated

Author: This description is not visible on the page. It is the front matter and this description is used to display the description on the Getting Started page.

Reviewer: ok

```bash
osdsctl fileshare delete <fileshare id>
```

### SODA Dashboard UI

SODA Dashboard UI is available at `http://{your_host_ip}:8088`, please login the dashboard using the default admin credentials: `admin/opensds@123.` Create `tenant`, `user`, and `profiles` as admin. Multi-Cloud service is also supported by dashboard.

### Please Note
{{% notice info %}}
**_To use the multicloud service, an AK/SK must be generated before anything else. To do this you can follow the steps below:_**  <br />
1. Go to AK/SK Management<br />
2. Click on Add AK/SK button.<br />
3. Save the file (Do not forget to save this file and keep it safe.)<br />
{{% /notice %}}

{{% notice info %}}

> **_To use the Block and File service a profile must be created before anything else. To do this you can follow the steps below:_**  <br />

Reviewer: ```suggestion
**_To use the Block and File service, respective profile must be created before using these services. To do this you can follow the steps below:_**  <br />
```

1. Go to Profiles<br />
2. Click on Create button.<br />

> 3. Enter the details and select the Storage type as Block or File. Create one profile for Block type and one for File.<br />

Reviewer: Please mention the profile names must be default_block or default_file for block and file, respectively

Author: Will make the change.
